internal/config/config.go

```diff
@@ -272,18 +272,20 @@ func (c *Config) SetJiraToken(token *oauth1.Token) {
 
 // configFile is a serializable representation of the current Viper configuration.
 type configFile struct {
-	LogLevel    string        `json:"log-level" mapstructure:"log-level"`
-	GithubToken string        `json:"github-token" mapstructure:"github-token"`
-	JiraUser    string        `json:"jira-user" mapstructure:"jira-user"`
-	JiraToken   string        `json:"jira-token" mapstructure:"jira-token"`
-	JiraSecret  string        `json:"jira-secret" mapstructure:"jira-secret"`
-	JiraKey     string        `json:"jira-private-key-path" mapstructure:"jira-private-key-path"`
-	JiraCKey    string        `json:"jira-consumer-key" mapstructure:"jira-consumer-key"`
-	RepoName    string        `json:"repo-name" mapstructure:"repo-name"`
-	JiraURI     string        `json:"jira-uri" mapstructure:"jira-uri"`
-	JiraProject string        `json:"jira-project" mapstructure:"jira-project"`
-	Since       string        `json:"since" mapstructure:"since"`
-	Timeout     time.Duration `json:"timeout" mapstructure:"timeout"`
+	LogLevel    string        `json:"log-level,omitempty" mapstructure:"log-level"`
+	DryRun      string        `json:"dry-run,omitempty" mapstructure:"dry-run"`
+	GithubToken string        `json:"github-token,omitempty" mapstructure:"github-token"`
+	JiraUser    string        `json:"jira-user,omitempty" mapstructure:"jira-user"`
+	JiraPass    string        `json:"jira-pass,omitempty" mapstructure:"jira-pass"`
```

Comment:
Are we sure we want to include passwords in plaintext?
ref: https://github.com/uwu-tools/gh-jira-issue-sync/issues/34

Reply:
This change is only about updating the user's config file, which the tool does after every run, it has nothing to do with the format of the password. It's always provided by the user as plain text but if we are going to remove username+password as an auth option then this shouldn't be a problem.

Reply:
Fair point. Let's keep that for now then!